dyno-jedis/src/main/java/com/netflix/dyno/jedis/OpName.java

```diff
@@ -22,8 +22,9 @@
 	 DECR, DECRBY, DEL, DUMP, 
 	 ECHO, EXISTS, EXPIRE, EXPIREAT, EVAL,
 	 GET, GETBIT, GETRANGE, GETSET, 
-	 HDEL, HEXISTS,  HGET, HGETALL, HINCRBY, HINCRBYFLOAT, HKEYS, HLEN, HMGET, HMSET, HSET, HSCAN, HSETNX, HVALS, 
-	 INCR, INCRBY, INCRBYFLOAT, 
+	 HDEL, HEXISTS,  HGET, HGETALL, HINCRBY, HINCRBYFLOAT, HKEYS, HLEN, HMGET, HMSET, HSET, HSCAN, HSETNX, HVALS,
+	 EHSET, EHGET, EHDEL, EHEXISTS, EHGETALL, EHKEYS, EHMGET, EHVALS, EHMSET, EHSETNX, EHSCAN, EHINCRBY, EHINCRBYFLOAT,
```

Comment:
Are these custom dynomite specific commands?

Reply:
These commands are dyno specific and build on top of existing Jedis commands.